Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationFailureRepository.cs

```diff
@@ -83,6 +83,11 @@ public bool HasValidationFailures ()
         return _rowFailures is { Count: > 0 } || _cellFailures.Count > 0 && _cellFailures.Values.SelectMany(e => e).Any();
       }
 
+      public bool HasUnhandledValidationFailures ()
```

Comment:
earlier, we moved all "Unhandled" logic to the outer class. please check if this works here, too.

Reply:
This functionality is not required after all, so I'll just remove it.